### ISSUE TITLE
Update site upload strategy isn't so great

### ISSUE THREAD

**Issue by kephale** [scenerygraphics/sciview#117]
When using the proper ImageJ release-versions.sh script, the version gets incremented and immediately promoted to the next SNAPSHOT, but the SciView trigger for upload to update site is triggered by a git commit message. With the current approach we either have to backtrack or upload SNAPSHOTs.

**Comment by imagejan** [comment]
`release-version.sh` does create a commit that sets the release version (see https://github.com/scenerygraphics/sciview/commit/f9aadd04c06ec984841b45a43d8623e548246f37), but it is not part of the `master` branch after the release was cut. Can we maybe automatically trigger the upload by that release commit?

**Comment by ctrueden** [comment]
> Can we maybe automatically trigger the upload by that release commit?

Yes, my suggestion would be for the SciView `.travis/build.sh` script to be appended to include additional logic that happens only for release tag builds.
